scripts/scrabble.js

```diff
@@ -0,0 +1,48 @@
+const rl = require( 'readline-sync')
+
+getInput = () => {
+  let theLetters = Array.from( rl.question( 'What are the letters? ' ).toUpperCase() )
+  return theLetters
+}
+
+let bagOfLetters =
+{
+  "E": 12,
```

Comment:
Dynamic keys tend to be a little more difficult to work with, depending on the way that the `bagOfLetters` object is going to be used.  I would consider using an array of "`Tile`" objects, where they keys are consistent and intention revealing:
```js
const bagOfLetters = [
  { letter: 'E', frequency: 12 },
  // etc.
]
```
Note that I also changed the `let` to a `const`, since the array should not be mutated (even though objects contained in the array may be mutated).